### ISSUE TITLE
Training details

### ISSUE THREAD

**Issue by fingoldo** [kurapan/CRNN#10]
Hi Jan, thank you so much for your work!

I'm new to OCR, but trying to use keras-ocr to solve a particular plastic cards text recognition problem. keras-ocr is using crnn_kurapan.h5 weights for its Recognizer, accuracy is noticeable, but still worse than accuracy of Amazon Rekognition, not speaking of ClovAI. So I decided to fine-tune model a bit, using ~1000 random backgrounds and ~2400 fonts which come with keras-ocr, and additionally applying a diverse set of albumentations transforms (except geometric, because albumentations library only supports rectangular bounding boxes). Texts generation of keras-ocr was modified: i was sampling its original texts 50% of time, rest of times it was a random subset of words mostly used in images of my main task.
But it turned out that, despite clearly improving both training and validation losses on such synthetic datataset, such fine-tuning ended up in a recognizer accuracy far worse than accuracy of your original weights on my main task.
Have you seen similar cases in your practice? Maybe you have some advice for me..

Also, can you share some training details for crnn_kurapan.h5, please? 
Was it trained on Synth90k only?
How long did it take you to train it, what was the hardware, what (approximately) accuracy metrics did you achieve? Was it possible to achieve more? 
I'm trying to understand what I need to do to improve its accuracy on my task without additional manual labelling. 
Also what if I would like to use extended alphabet (+#='.:, capitals), clearly Synth90k would not be a good fit anymore?
I have one Tesla V100 GPU. Your advice would be very appreciated!
